content/en/blog/_posts/2020-04-06-API-Priority-and-Fairness-Alpha.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog
+title: "API Priority and Fairness Alpha"
+date: 2020-04-06
+slug: kubernetes-1-18-feature-api-priority-and-fairness-alpha
+---
+
+**Authors:** Min Kim (Ant Financial), Mike Spreitzer (IBM), Daniel Smith (Google)
+
+This blog describes “API Priority And Fairness”, a new alpha feature in Kubernetes 1.18. API Priority And Fairness permits cluster administrators to divide the concurrency of the control plane into different weighted priority levels. Every request arriving to kube-apiserver will be categorized into one of the priority levels and get its fair share of the control plane’s throughput.
+
+## What problem does this solve?
+Today the apiserver has a simple mechanism for protecting itself against CPU and memory overloads: max-in-flight limits for mutating and for read-only requests. Apart from the distinction between mutating and readonly, no other distinctions are made among requests; consequently, there can be undesirable scenarios where one subset of the requests crowds out other requests.
```

Comment:
We probably should take a consistent choice on "readonly" vs "read-only".

Reply:
Maybe also “API server” vs “apiserver”?

Reply:
I updated to readonly, since that was used more often. apiserver was already in use throughout the document so left it at that.